components/BottomSheet/src/private/MDCSheetContainerView.m

```diff
@@ -215,14 +249,20 @@ - (CGFloat)truncatedPreferredSheetHeight {
 
 // Returns the maximum allowable height that the sheet can be dragged to.
 - (CGFloat)maximumSheetHeight {
-  CGSize size = self.bounds.size;
-  CGFloat scrollViewContentHeight = self.sheet.scrollView.contentSize.height;
+  CGFloat boundsHeight = self.bounds.size.height;
```

Comment:
CGRectGetHeight() since bounds may not be standardized.

Reply:
Done (we were already using self.bounds.size.height fwiw).
PTAL

Reply:
Indeed, we definitely haven't been as strict as we should have been in the past, but if we are touching the code.....